### PR TITLE
02-numpy.md: Remove array arithmetic

### PR DESCRIPTION
**Pull request by ldko**
This removes the portion of the Analyzing Patient Data episode that teaches common arithmetic operations with arrays. Doing this helps to shorten the episode by removing content that is not used as the lesson progresses.

### CODE REVIEW

File: _episodes/02-numpy.md [hunk 1]
```diff
@@ -319,66 +319,9 @@ small is:
...
-tripledata:
-[[ 6.  9.  0.  0.]
- [ 3.  3.  0.  3.]
- [ 6.  6.  3.  3.]]
-~~~
-{: .output}
-
-Often, we want to do more than add, subtract, multiply, and divide array elements.  NumPy knows how
-to do more complex operations, too.  If we want to find the average inflammation for all patients on
+## Analyzing data
```
Comment (maxim-belkin) [inline _episodes/02-numpy.md]: Please add an empty line after this one

**Comment by maxim-belkin** [comment]
Changes look good to me -- thanks, Lauren!

I have a few comments about the sentences in this section but I think we can address them separately.

File: _episodes/02-numpy.md (hunk 1 above)
Comment (ldko) [reply _episodes/02-numpy.md]: The blank line has been added

Comment (maxim-belkin) [reply _episodes/02-numpy.md]: Thank you!

**Pull request merged**